core/tracepoint/enable_spec.rb

```diff
@@ -96,6 +96,55 @@
     end
   end
 
+  describe 'no options' do
+    before :each do
+      ScratchPad.record []
+    end
+
+    it 'traces method call' do
+      klass = Class.new do
+        def foo
+          bar
+        end
+        def bar; end
+      end
+
+      trace = TracePoint.new(:call) do |tp|
+        ScratchPad << tp.method_id
+      end
+
+      obj = klass.new
+
+      trace.enable do
+        obj.foo
+      end
+
+      ScratchPad.recorded.should == [:foo, :bar]
+    end
+
+    ruby_bug '#16383', '2.4'...'2.8' do
```

Comment:
Current MRI master reports as 2.8 but doesn't fix this bug, so this should be at least 3.0.

Reply:
OK, I changed 2.8 to 3.0